### PR TITLE
#P04.1. Only display published Hackathons

### PR DESCRIPTION
**Pull request by natalijabujevic0708**
## Description
Added an if statement on hackathon_list.html to show only Hackathons with status published

## Pull request type
#P04.1. Only display published Hackathons

## Screenshots

BEFORE:
before

AFTER:
after

## Does this introduce a breaking change

- [ ] Yes
- [X] No

### CODE REVIEW

File: hackathon/templates/hackathon/hackathon_list.html [hunk 1]
```diff
@@ -2,18 +2,20 @@
...
-        <h2 class="text-center">There aren't any Hackathons at the moment!</h2>
-    {% endif %}
-    <!-- Pagination using Django's paginator. Options are disabled or hidden if unavailable. -->
-    <nav aria-label="Hackathon Page Navigation">
-        {% include 'hackathon/includes/paginator.html' %}
-    </nav>
+	{% if page_obj %}
+	<h2 class="p-orange text-center mb-4">Hackathons</h2>
+	{% for hackathon in page_obj %}
+        {%if hackathon.status == 'published' %}
```
Comment (stefdworschak) [inline hackathon/templates/hackathon/hackathon_list.html:8]: Could you change this to be either `published` or `draft` and created by the current user?

**Review by stefdworschak — CHANGES_REQUESTED** [review]
@natalijabujevic0708 please have a quick look at my comment and let me know once I can approve it.

**Comment by stefdworschak** [comment]
Also, it seems you have a merge conflict. Please resolve that before reverting back to me.

**Comment by natalijabujevic0708** [comment]
@stefdworschak I did the changes and tested it. So I created 2 Hackathos with status draft, one created by me and other by admin. At the site I could only see the draft created by me, and a published Hackathon created by admin!
admin
display

**Review by stefdworschak — APPROVED** [review]
LGTM

**Pull request merged**